crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -508,6 +509,17 @@ impl Room {
         self.inner.read().base_info.max_power_level
     }
 
+    /// Gets this Room's power levels.
+    pub async fn power_levels(&self) -> Option<RoomPowerLevels> {
+        self.store
+            .get_state_event_static::<RoomPowerLevelsEventContent>(&self.room_id)
+            .await
+            .ok()
+            .flatten()
+            .and_then(|raw| raw.deserialize().ok())
+            .map(|ev| ev.power_levels())
+    }
+
```

Comment:
We already have `matrix_sdk::Room::room_power_levels`, but we needed something at the `matrix_sdk_base` crate too. Since `sdk::Room` already has access to the fns in `sdk_base::Room`, should I just remove the one in sdk and use this one instead?

Reply:
Indeed, they do the same thing. Can we have this function here return a `Result` instead, to not silently swallow errors, please? (and we can keep the shorter `power_levels` name, I like it better :+1:)